src/khoj/interface/web/khoj_chat.webmanifest

```diff
@@ -5,7 +5,7 @@
     "icons": [
         {
             "src": "/static/assets/icons/favicon-144x144.png",
```

Comment:
Should we just rename the new favicon to favicon-128x128.png? Or increase it's size to 144x144? Having a 128x128 image in a file called favicon-144x144.png maybe confusing 😅

Reply:
Haha yeah, my bad. I was trying to migrate it in-place to test out before changing all the references. Updating.